### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by subin-kim-cv** [NIRVANALAN/E3DGE#1]
Hello, thanks for your awesome work.

I'm a little bit confused about the training details.

1. Train/test set on 2D reconstruction
Is StyleSDF pre-trained on Celeb-HQ  or FFHQ?

2. Train about Encoders E0 and E1. 
(a) training requires two encoders separately
(b) train two encoders jointly and, at some phase, freeze the weight of network E0

Could you kindly specify which one is correct between (a) and (b)

Many thanks!

**Comment by NIRVANALAN** [comment]
Hi Kim, thanks for your interest in our work.

1. StyleSDF is pre-trained on FFHQ (official checkpoint) and our method is test on CelebA-HQ test split.
2. (a) is correct, in stage 1 we train E0 so that a plausible shape inversion is guaranteed. Then, we train E1 to add texture details.

Feel free to let me know if more elaboration is needed!

Best regards,
Yushi
